feg/cloud/go/services/feg_relay/gw_to_feg_relay/servicers/session_proxy_relay.go

```diff
@@ -44,60 +45,85 @@ func (s *RelayRouter) UpdateSession(
 	// Group requests by PLMNID
 	// Use the longest possible PLMN ID (6), in the worst case we'll fragment requests more then needed, but the routing
 	// will still be logically correct
-	reqMap := map[string]*protos.UpdateSessionRequest{}
+	reqMap := map[string]map[gateway_registry.GwServiceType]*protos.UpdateSessionRequest{}
```

Comment:
lets add comments so we don't forget what we are doing here.

You can use something like
```
	// Group requests by [PLMNID][GwServiceType]
	// - PLMNID: Use the longest possible PLMN ID (6), in the worst case we'll fragment requests more then needed, but the routing will still be logically correct
	// - GwServiceType: each request will be sent to either Session_Proxy or for N7_N40_proxy
```

Reply:
Done